CHANGELOG.md

```diff
@@ -24,9 +24,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 - A warning about the current reducer API limitations, as well as a mention of active work to mitigate them was added to doc comments and examples https://github.com/o1-labs/o1js/pull/1728
-
 - `ForeignField`-based representation of scalars via `ScalarField` https://github.com/o1-labs/o1js/pull/1705
 - Introduced new V2 methods for nullifier operations: `isUnusedV2()`, `assertUnusedV2()`, and `setUsedV2()` https://github.com/o1-labs/o1js/pull/1715
+- `Int64.create()` method for safe instance creation with canonical zero representation https://github.com/o1-labs/o1js/pull/1735
+- New V2 methods for `Int64` operations: `fromFieldV2()`, `fromFieldUncheckedV2()`, `divV2()`, `isPositiveV2()` https://github.com/o1-labs/o1js/pull/1735
```

Comment:
nit: `isPositiveV2()` existed before this PR

Reply:
Fixed! https://github.com/o1-labs/o1js/pull/1735/commits/8e05a1546de47571b31ef7d922dcbc44673067b7